source/includes/_datasets.md

```diff
@@ -41,7 +41,7 @@ Through the management API, it is possible to:
 Datasets are identified by 2 kinds of identifiers:
 
 - the `dataset_uid` that is automatically set, and will never change through the lifetime of the dataset.
-- the `dataset_id`, that can be chosen during dataset creation or changed on an unpublished dataset
+- the `dataset_id`, that can be chosen during dataset creation or changed on an unpublished dataset. It will be used in the exploration API and UI to access the dataset with a meaningful URL.
```

Comment:
s/exploration/explore
I don't think I've seen "exploration" anywhere, we always say "explore" (that may be a bad practice but coherence etc.)

Reply:
👍